### DISCUSSION TITLE
api/auth/[...nextauth] is exceeding the maximum serverless function size limit of 50mb on Vercel

### DISCUSSION THREAD

**Discussion by swapnil233** [nextauthjs/next-auth#8159] [Help]
My Nextjs app is using NextAuth, and just recently I started getting an error on Vercel upon attempted deployment saying:

`
Error: The Serverless Function "api/auth/[...nextauth]" is 53.23mb which exceeds the maximum size limit of 50mb. Learn More: https://vercel.link/serverless-function-size
`

Any idea how to fix this? Here's my page data:

`
Route (pages)                                            Size     First Load JS
┌ ○ /                                                    1.45 kB         174 kB
├   /_app                                                0 B             130 kB
├ ○ /404                                                 1.14 kB         174 kB
├ λ /api/auth/[...nextauth]                              0 B             130 kB
├ λ /api/aws/getSignedUrl                                0 B             130 kB
├ λ /api/aws/presignedUploadUrl                          0 B             130 kB
├ λ /api/deepgram                                        0 B             130 kB
├ λ /api/embeddings                                      0 B             130 kB
├ λ /api/file/create                                     0 B             130 kB
├ λ /api/invitation/accept                               0 B             130 kB
├ λ /api/invitation/create                               0 B             130 kB
├ λ /api/projects                                        0 B             130 kB
├ λ /api/summaries                                       0 B             130 kB
├ λ /api/summarize                                       0 B             130 kB
├ λ /api/teams                                           0 B             130 kB
├ λ /api/transcribe                                      0 B             130 kB
├ λ /api/user/role                                       0 B             130 kB
├ λ /api/webhooks/deepgram                               0 B             130 kB
├ λ /api/webhooks/stripe                                 0 B             130 kB
├ λ /teams                                               9.21 kB         192 kB
├ λ /teams/[teamId]                                      13.3 kB         196 kB
├ λ /teams/[teamId]/projects/[projectId]                 15.5 kB         199 kB
└ λ /teams/[teamId]/projects/[projectId]/files/[fileId]  6.96 kB         180 kB
+ First Load JS shared by all                            131 kB
  ├ chunks/framework-2c79e2a64abdb08b.js                 45.2 kB
  ├ chunks/main-2d7a063da46561ed.js                      27.1 kB
  ├ chunks/pages/_app-cdf4666a49265335.js                56.4 kB
  ├ chunks/webpack-880e48e2fc817058.js                   1 kB
  └ css/30c533f365fb6a66.css                             1.34 kB
`

**Comment by Strajk** [comment]
This helped me https://github.com/orgs/vercel/discussions/103#discussioncomment-6676753
